src/test/java/com/buinak/imageapi/controller/ImageControllerIT.java

```diff
@@ -30,38 +34,67 @@
     @Autowired
     ImageRepository imageRepository;
 
-    final static String IMG_PATH = "src/test/java/com/buinak/imageapi/controller/testimg.jpeg";
+    @Autowired
+    ImageService imageService;
 
 
     @Test
-    public void addImage() throws IOException {
-        MultipartFile multipartFile = new MockMultipartFile("testimg.jpeg", new FileInputStream(new File(IMG_PATH)));
+    public void addImage() {
+        MockMultipartFile multipartFile = new MockMultipartFile("data", "testimg.jpeg", "text/plain", "some xml".getBytes());
```

Comment:
hm....... can't you just mock the storage service instead and verify the calls were successfully made? why mock a file?

Reply:
Use Mockito